src/cljs/nnts2/routes.cljs

```diff
@@ -18,10 +18,12 @@
 
 (defroute "/note" []
   (do
-    (re-frame/dispatch [::events/set-active-panel :note-panel])))
+    (re-frame/dispatch [::events/set-active-panel :note-panel])
+    (re-frame/dispatch [:note-get-list])))
```

Comment:
Would you not want to get a list of notes on `"/home"`?

Reply:
Also, seems like get-all-notes is called only upon notes creation. Wouldn't we want to retrieve and show all notes after the user logs in?

Reply:
It will be called whenever the notes panel is loaded. At the point, its not integrated with home panel.

Reply:
We will load the home panel when the user logs in